.travis.yml

```diff
@@ -7,6 +7,13 @@ jobs:
     - stage: test
       node_js: "12"
       os: linux
+      arch: amd64
```

Comment:
Please move the amd64 builds under the osx builds.

Reply:
Shouldn't they be grouped by stage?

Reply:
There should only be one `test` stage. It's poorly named as actually runs the linter.